pkg/filter/filter_test.go

```diff
@@ -453,6 +458,18 @@ func (s *FilterSuite) TestResourceIncludeExclude(c *C) {
 			include:   []Resource{ss2, pvc2},
 			exclude:   []Resource{ss1, pvc1},
 		},
+		{
```

Comment:
Need more unit tests here to demonstrate usage and corner cases.  For example, test to filter only on label.

Reply:
Added two tests which required a modification to the Matches logic.